src/NexusMods.App.UI/Pages/NexusModsDataProvider.cs

```diff
@@ -165,7 +175,7 @@ public IObservable<IChangeSet<LoadoutItemModel, EntityId>> ObserveNestedLoadoutI
 
                 var installedAtObservable = observable
                     .Transform((_, e) => LibraryLinkedLoadoutItem.Load(_connection.Db, e).GetCreatedAt())
-                    .QueryWhenChanged(query => query.Items.FirstOrDefault());
+                    .QueryWhenChanged(query => query.Items.Max());
```

Comment:
This causes exception on deletes, as it can become an empty collection. Should check if Items is empty and return the default in that case.